.ci/scripts/utils.sh

```diff
@@ -62,10 +62,38 @@ install_pytorch_and_domains() {
   git checkout "${TORCH_VERSION}"
   git submodule update --init --recursive
 
-  export USE_DISTRIBUTED=1
-  # Then build and install PyTorch
-  python setup.py bdist_wheel
-  pip install "$(echo dist/*.whl)"
+  SYSTEM_NAME=$(uname)
+  if [[ "${SYSTEM_NAME}" == "Darwin" ]]; then
+    PLATFORM=$(python -c 'import sysconfig; import platform; v=platform.mac_ver()[0].split(".")[0]; platform=sysconfig.get_platform().split("-"); platform[1]=f"{v}_0"; print("_".join(platform))')
+  fi
+  PYTHON_VERSION=$(python -c 'import platform; v=platform.python_version_tuple(); print(f"{v[0]}{v[1]}")')
+  TORCH_RELEASE=$(cat version.txt)
+  TORCH_SHORT_HASH=${TORCH_VERSION:0:7}
+  TORCH_WHEEL_PATH="cached_artifacts/pytorch/executorch/pytorch_wheels/${SYSTEM_NAME}/${PYTHON_VERSION}"
+  TORCH_WHEEL_NAME="torch-${TORCH_RELEASE}%2Bgit${TORCH_SHORT_HASH}-cp${PYTHON_VERSION}-cp${PYTHON_VERSION}-${PLATFORM:-}.whl"
```

Comment:
should we have local variables instead of global env variables

local system_name, torch_release etc?

Reply:
Just FYI, after updating these variables with `local`, I look around and find some feature request on shellcheck about this https://github.com/koalaman/shellcheck/issues/468, but it hasn't been implemented yet (probably not anytime soon)